Entities/Structures/Platform/WoodenPlatform.as

```diff
@@ -12,14 +12,23 @@ void onInit(CBlob@ this)
 	CShape@ shape = this.getShape();
 	shape.AddPlatformDirection(Vec2f(0, -1), 89, false);
 	shape.SetRotationsAllowed(false);
-
+	
 	this.server_setTeamNum(-1); //allow anyone to break them
-
 	this.set_TileType("background tile", CMap::tile_wood_back);
-
 	this.set_s16(burn_duration , 300);
 	//transfer fire to underlying tiles
 	this.Tag(spread_fire_tag);
+	
+	if (this.getName() == "wooden_platform")
```

Comment:
Not 100% on the purpose of the name check here and in the ladder; aren't these single-blob scripts?